### PR TITLE
fix: #73

### LINKED ISSUE #73: Bot also replies in dm
### Describe the bug

the bot will reply on the command even if its dm

### To Reproduce

Dm the bot `cm!help`
It will reply

### Expected Behavior

_No response_

### Screenshot/ Video

https://tanshadow.is-inside.me/JNGa5Jqn.png

### Desktop (please complete the following information)

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by krishguptadev**
## Fixes Issue

This PR fixes the following issues:

fixes #73 

## Changes proposed

Here comes all the changes proposed through this PR

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots 
Add all the screenshots which support your changes

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -40,9 +40,9 @@ client.on('ready', () => {
 
 client.on('message', (message) => {
 	if (
-		!message.content.startsWith(prefix) ||
+		(!message.content.startsWith(prefix)) ||
 		message.author.bot ||
-		message.channel.type == 'dm'
+		message.channel.type == 'DM'
```
Comment (kaiwalyakoparkar) [inline index.js:45]: Could you please explain the reason behind this update?

Comment (krishguptadev) [reply index.js:45]: For some reason it was replying in dms... I checked the docs and they have uppercase specifications for this... so I thought to fix the bug...

Comment (kaiwalyakoparkar) [reply index.js:45]: But we want the replies in DM :thinking:

Comment (krishguptadev) [reply index.js:45]: check my comment on the original issue

Comment (kaiwalyakoparkar) [reply index.js:45]: Ah, got it. So will this fix the issue in #73 ?

Comment (krishguptadev) [reply index.js:45]: Ah Got it the error is not this.. I found the error I'll make another commit to this pr... that should fix it

Comment (krishguptadev) [reply index.js]: Now it doesn't reply in the dms.. Tested. For some reason not enclosing it into brackets was the bug

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM

**Pull request merged**